webclient/app/src/app/pages/appEditor/sections/erSection/ErSection.jsx

```diff
@@ -115,23 +79,14 @@ function ErDesigner(props) {
         entity.position.positionX = draggableData.x;
         entity.position.positionY = draggableData.y;
 
-        entityRest.updateEntityByAppId(appId, entity)
-
-        updateCoordinates(entities);
-    }
-
-    function renderRelations() {
-
-        return coordinates.map((coordinate, index) => {
-            return <SteppedLineTo from={coordinate.from} to={coordinate.to} borderColor={theme.palette.primary.main} borderWidth={theme.palette.line.width} />
-        })
+        entityRest.updateEntityByAppId(appId, entity);
```

Comment:
The updated data from the server is not injected into the state. Maybe a update of the state from the server would be great here